skills/declarative/calendarSkill/dialogs/GetContactDialog/language-generation/en-us/GetContactDialog.en-us.lg

```diff
@@ -1,19 +1,22 @@
 [import](common.lg)
 [import](contactList.en-us.lg)
 
-# SendActivity_UrRzOu()
-[Activity
-    text = I could not find the exact attendee you were looking for.
-    attachments = ${json(NoContactsCard())}
-]
-
 # TextInput_Prompt_wIEu2k()
+- IF: ${ dialog.tryAgain == true }
+    - ${AskForAttendeeTryAgain()}
+- ELSE:
+    - ${AskForAttendee()}
+
+# AskForAttendee()
 - Who will join?
 - Who do you want to invite?
 - What attendees do you want to add?
 
-# SendActivity_mD3TcZ()
-- If you want to try again, just call out the first name and or surname again.
+# AskForAttendeeTryAgain()
+[Activity
+    text = I could not find the exact attendee you were looking for.
+    suggestedActions = Proceed without attendees
```

Comment:
SuggestedActions don't work in Teams today, so we'll need to modify this to be a hero card. I'll make the change